docs/application/dotnet/guides/nui/styling-controls-with-xaml.md

```diff
@@ -0,0 +1,201 @@
+# Styling UI components with XAML
+Styling of components in NUI is possible in two ways:
+  - `ViewStyle`: designed for simple themes.
+  -  XAML file: designed for advanced themes. With XAML file, styles change in runtime and can be shared between the applications.
+
+## ViewStyle theme
+The basic styling of NUI components is based on the `ViewStyle` class and its derivatives. `ViewStyle` defines attributes and stores their values. 
+
+The following is a list of the most common attributes:  
+- name
+- state 
+- subState
+- position
+- scale
+- orientation
+- padding 
+
+You can find all the attributes and their details on [ViewStyle attributes](https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs) page.
+
+In addition, styles can also be defined for the components with following states:
+| State              | Description                                                                                                                                          |
+|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Normal`           | Specifies the default style.                                                                                                                         |
+| `Focused`          | Specifies the state where all the keystrokes and events are recieved by the widget. There can only be one widget in the active window at a time in this state. |
+| `Disabled`         | Specifies the state where the widget is disabled. All other events are ignored in this state.                                                          |
+| `Selected`         | Specifies the state where a widget is selected.                                                                                                      |
+| `Pressed`          |  Specifies the state where the widget receives touch event.                                                                                                                                                   |
+| `DisabledFocused`  | Specifies the state that is similar to the `Focused` state, but there is no possibility to change the widget state.                                                                        |
+| `SelectedFocused`  | Specifies the state where the widget is checked and all events are received.                                                                                            |
+| `DisabledSelected` | Specifies the combination of disabled and selected states.                                                                                                       |
```

Comment:
**Original :** specifies the state where all the keystrokes and events reach the widget. There can be only one widget in the active window at a time in this state.
**Suggestion :** Specifies the state where all the keystrokes and events are recieved by the widget. There can only be one widget in the active window at a time in this state


@annie-samsung Suggestion. How about reducing this kind of review that just tweak the sentence slightly?
In here, over 170 comments were added, which means lots of points were updated.
However errors such as wrong target branch(6.5) or missing semicolon of end of the code were not checked.

As I asked several times, I hope you to focus on the grammar and context errors.
There are only two review members in SRI-B, so more effective review is needed.
For now, comments just to tweaking the sentences slightly & cross checking that huge amount of comments drains our much time and energy, but it is less effective at fixing the errors in the content.

Reply:
@mijin85cho  I do understand your concern, and I do agree with your statements. I usually keep Grammar, structure, and consistency as primary review criteria. My motive is always to make the text easier to read and thus easy to comprehend. There are 170+ comments, only when I feel things need tweaking for making better sense. As you rightly suggested removing specifies, definitely increases readability. I reframed the whole table with your suggestion. Thats what I too usually do.

Reply:
I missed the branch thing, but Annie had pointed it out. 
 https://github.com/Samsung/tizen-docs/pull/1386/files#r574320373

Reply:
> My motive is always to make the text easier to read and thus easy to comprehend. 

@nawab-samsung @annie-samsung The problem is that this rewriting is often wrong in the meaning, or creates the grammatically wrong sentence, and makes everyone exhausted with never-ending review. For example this PR was created last year and 26 updates were made, but as I said some obvious issues were not fixed even after more than 170 comments. Because of it, not rewriting and only fixing errors seems to be more efficient, so I'm asking again and again not to rewrite the sentences.

Reply:
@mijin85cho, I understand your concern, let us discuss offline how can we make it easier for everyone(Developer, Tech. Writer, Editor(final reviewer)).